contracts/DCAStrategies/DCAStrategies/DCAStrategiesPermissionsHandler.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol';
+import '../../interfaces/IDCAStrategies.sol';
+
+abstract contract DCAStrategiesPermissionsHandler is IDCAStrategiesPermissionsHandler, ERC721Enumerable {
+  // solhint-disable-next-line func-name-mixedcase
+  function PERMIT_TYPEHASH() external pure override returns (bytes32) {}
+
+  // solhint-disable-next-line func-name-mixedcase
+  function PERMISSION_PERMIT_TYPEHASH() external pure override returns (bytes32) {}
+
+  // solhint-disable-next-line func-name-mixedcase
+  function PERMISSION_SET_TYPEHASH() external pure override returns (bytes32) {}
+
+  // solhint-disable-next-line func-name-mixedcase
+  function DOMAIN_SEPARATOR() external view override returns (bytes32) {}
+
+  function nftDescriptor() external override returns (IDCAHubPositionDescriptor) {}
```

Comment:
We will use a different descriptor here, so could you add a `TODO` comment that we need to update this when we build said descriptor?